pkg/embeddedyamls/yamls.go

```diff
@@ -0,0 +1,3151 @@
+/*
```

Comment:
now that this file is available, we can remove the logic to generate them on various `make` invocations.

Reply:
We still need to ensure it’s always up-to-date; I’d rather leave the prerequisites as-is in the Makefile.